src/ucp/core/ucp_ep.c

```diff
@@ -1028,8 +1028,12 @@ static void ucp_ep_config_calc_params(ucp_worker_h worker,
     memset(params, 0, sizeof(*params));
 
     for (i = 0; (i < UCP_MAX_LANES) && (lanes[i] != UCP_NULL_LANE); i++) {
-        lane       = lanes[i];
-        rsc_index  = config->key.lanes[lane].rsc_index;
+        lane      = lanes[i];
+        rsc_index = config->key.lanes[lane].rsc_index;
+        if (rsc_index == UCP_NULL_RESOURCE) {
```

Comment:
it is possible in case of error handling:
`ucp_worker_iface_err_handle_progress()` - fills lanes by `0` and `rsc_index` by `UCP_NULL_REOSURCE`
  |
\\/
`ucp_worker_get_ep_config`
  |
\\/
`ucp_ep_config_init`
|
\\/
`ucp_ep_config_calc_rma_zcopy_thresh`
|
\\/
`ucp_ep_config_calc_params`

Reply:
ok